elements/ol.html

```diff
@@ -0,0 +1,45 @@
+<article class="article">
+  <header class="article-header">
+    <h2>
+      <code>
+        &lt;ol&gt;
+      </code>
+    </h2>
+    <div class="base-words">
+      <div class="base-word-line">
+        English Base: <strong>Ordered List</strong>
+      </div>
+      <div class="base-word-line">
+        Spanish translation/Traducción en Español: <strong></strong>
+      </div>
+    </div>
+  </header>
+  <div class="definition">
+    <div class="definition-box definition-spanish">
+      <h4>Definici&oacute;n</h4>
+      <p></p>
+    </div>
+    <div class="definition-box definition-english">
+      <h4>Definition</h4>
+      <p>Used to group a list of items in order with a preceding number, letter or Roman numeral.  The preceding number system is indicated in the CSS file using list-style-type to choose between using numbers, letters or Roman numerals, however the actual starting number is in the html file using the start attribute.</p>
```

Comment:
Reduce some duplication to shorten and separate into different sentences.  I prefer the oxford comma.

Used to group a list of items in order with a preceding number, letter**,** or Roman numeral.  The preceding number system is indicated in the CSS file using list-style-type**. T**he actual starting number is in the html file using the start attribute.